RLTest/__main__.py

```diff
@@ -310,10 +310,11 @@ def __init__(self):
             br.download_binaries()
 
         if self.args.clear_logs:
-            try:
-                shutil.rmtree(self.args.log_dir)
-            except Exception as e:
-                print(e)
+            if os.path.exists(self.args.log_dir):
+                try:
+                    shutil.rmtree(self.args.log_dir)
+                except Exception as e:
+                    print(e)
```

Comment:
`print(e, file=sys.stderr)`?
We can also consider `shutil.rmtree(..., ignore_errors=True)` but I'm not sure I trust this thing (py2/3 you know).

Reply:
Why are we not using the standard python logging? it makes this things easier with `log.exception`.

Reply:
fixed in the last commit